command.go

```diff
@@ -1119,7 +1122,13 @@ func (c *Command) ExecuteC() (cmd *Command, err error) {
 		// Always show help if requested, even if SilenceErrors is in
 		// effect
 		if errors.Is(err, flag.ErrHelp) {
-			cmd.HelpFunc()(cmd, args)
```

Comment:
The use of `args` here was added in this [commit](https://github.com/spf13/cobra/pull/138/commits/c2d19e9c4338e36a2f2176537a4f67b3ee59f4e7) and it does seem that it was a mistake; I believe the `flags` variable should have been used instead.